src/test/java/pl/kl/weatherservice/location/LocationsControllerIntegrationTest.java

```diff
@@ -0,0 +1,70 @@
+package pl.kl.weatherservice.location;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.mock.web.MockHttpServletResponse;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockHttpServletRequestBuilder;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+public class LocationsControllerIntegrationTest {
+
+    @Autowired
+    MockMvc mockMvc;
+    @Autowired
+    LocationsService locationsService;
+    @Autowired
+    LocationsRepository locationsRepository;
+
+    @Test
+    void postLocation_createsNewLocation() throws Exception {
+        // given
+        locationsRepository.deleteAll();
+        ObjectMapper objectMapper = new ObjectMapper();
+        CreateLocationRequest requestBody = LocationTestHelper.LOCATION_REQUEST_GDANSK;
```

Comment:
let's use a methods instead of static variables (each test can change an internal state of this object)

Reply:
Fixed. Used static method.